cnap/core/keybroker.py

```diff
@@ -68,13 +70,14 @@ class SimpleKeyBrokerClient(KeyBrokerClientBase):
           key (SWK) to encrypt the user key (wrapped_key).
         - Encrypt the SWK by the public key from client (wrapped_swk).
     For a key broker client, here is an example flow to get a key from KBS:
+        - Verify Event Logs.
```

Comment:
=> Get and replay all event logs, and verify by the measurement register.

Reply:
Thanks, fixed.